pkg/endpoint/policy.go

```diff
@@ -16,6 +16,7 @@ package endpoint
 
 import (
 	"fmt"
+	"github.com/cilium/cilium/pkg/uuid"
```

Comment:
import order

Reply:
Sigh, sorry for the noise. I think I messed up my automatic goimports format in my IDE...